SwitchSpeak-iOS/CrumbManagerTest/CrumbManagerTest.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CrumbManagerTest.swift
+//  CrumbManagerTest
+//
+//  Created by Ning Luo on 2018/3/25.
+//  Copyright © 2018年 SwitchSpeak. All rights reserved.
+//
+
+import XCTest
+@testable import SwitchSpeak_iOS
+
+class CrumbManagerTest: XCTestCase {
+    var CrumbManagerTest:CrumbStack!
+    override func setUp() {
+        CrumbManagerTest = CrumbStack()
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        CrumbManagerTest=nil
+        super.tearDown()
+    }
+    
+    func emptyCrumstacktest() {
+        CrumbManagerTest.push(string: "hello test1")
+        CrumbManagerTest.push(string: "hello test2")
+        CrumbManagerTest.emptyCrumbStack()
+       // let x = CrumbManagerTest.getString()
+        XCTAssertNil(CrumbManagerTest.getString())
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+    
+    func CrumStackPushTest()  {
+        let x1: String = "hello test for push1"
+        let x2: String = "hello test for push2"
+        CrumbManagerTest.push(string: x1)
+        CrumbManagerTest.push(string: x2)
+        let content = x1+" "+x2
+        XCTAssertEqual(CrumbManagerTest.getString(), content)
+    }
+    
+    func CrumbUpdateSubviewTest() {
+        let TestView = UIView()
+        let ContainerView = UIView()
```

Comment:
Please rename these variables so that they are in camelCase. i.e. `containerView` instead of `ContainerView`.